sample_project/Assets/SampleViewer/Scripts/TouchCameraController.cs

```diff
@@ -0,0 +1,11 @@
+using UnityEngine;
+
+public class TouchCameraController : MonoBehaviour
```

Comment:
This name implies this would be doing something with the canvas, not just toggling UI.